app/views/admin/teachers/_form.html.erb

```diff
@@ -1,7 +1,11 @@
-<%= simple_form_for([:admin, resource], html: { class: 'inline' }) do |form| %>
-  <%= render 'error_notification', form: form %>
-  <%= form.input :name %>
-  <%= form.input :email %>
-  <%= form.select :grove_id, collection: current_user.groves.map{|x| [x.name, x.id]} %>
-  <%= form.submit "Save" %>
-<% end %>
+<div class='container'>
+  <%= simple_form_for([:admin, resource], html: { class: 'inline' }) do |form| %>
+    <%= render 'error_notification', form: form %>
+    <%= form.input :name %>
+    <%= form.input :email %>
+    <%= form.select :grove_id, collection: current_user.groves.map{|x| [x.name, x.id]} %>
+    <%= form.label 'Is Admin?' %>
+    <%= form.input_field :role_ids, as: :boolean %>
+    <%= form.submit 'Save' %>
```

Comment:
Could you think of perhaps using something like form.associations for flexibility with roles? https://github.com/plataformatec/simple_form#associations